source/get-commits.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
```

Comment:
Extracted from `ui.js`, because it's also used in `detect-version.js`.

Reply:
I'd move this to `git-util.js`, as this is mainly git-related and belongs there.